R/install.R

```diff
@@ -632,7 +678,7 @@ wrap_command <- function(x) {
 }
 
 set_envvar <- function(envs, action = "replace") {
-  stopifnot(is.named(envs))
+  stopifnot(all.named(envs))
```

Comment:
Could you add a comment explaining the behavior of this function (and also to `with_envvar`? The behavior of `UNSET_ME=NA` is a little subtle.